beetsplug/missing.py

```diff
@@ -204,10 +252,10 @@ def _missing_albums(self, lib, query):
             if total:
                 continue
 
-            missing_titles = {rg['title'] for rg in missing}
-
-            for release_title in missing_titles:
-                print_(u"{} - {}".format(artist[0], release_title))
+            for album in missing:
+                for album_info in hooks.albums_for_id(album['id']):
+                    item = _album_item(album_info, album_info.album_id)
```

Comment:
Sorry for being slow, but I don't quite understand what the deal is with the new `_album_item` filter. Can you explain a little more how this works?

Reply:
To use the `format` function I needed an object previously returned by `_item`, but `_item` expects a `track_info` object.
Because this is applied to albums I don't have a `track_info` object to pass to `_item`, hence I created a new filter called `_album_item` that accept an `album_info`.